sru-templates/manual/sru-vars.rc

```diff
@@ -0,0 +1,104 @@
+# This file is sourced by any manual SRU cloud tests
+# It should be stored in $(CWD)sru-vars.rc
+# Set the variables accordingly for your cloud configuration and usernames
+
+PRESERVE_INSTANCE=false   # can be set as cmdline param
+UBUNTU_RELEASE=""         # The release to test specfied as cmdline param
+LP_USER="UNSET"            # specify your launchpad username
+
+USE_DEV_PPA=0  # set to 1 to use ppa:cloud-init-dev/proposed
+if [ "${USE_DEV_PPA:-0}" -eq "0" ]; then
+    PROPOSED_SCRIPT=setup_proposed.sh
+else
+    PROPOSED_SCRIPT=setup_dev_proposed.sh
+fi
```

Comment:
This logic looks incomplete to me; the Azure test hasn't been updated to understand what `setup_dev_proposed.sh` is. Perhaps this would be better implemented by using the same filename for either version, and simply writing different content within `create_setup_proposed_scripts`?

Reply:
That makes sense. The azure script just scp's $PROPOSED_SCRIPT to $VM_IP:.  It probably it simpler just to leave the file_named the same and just render the appropriate content inside depending on USE_DEV_PPA 0 || 1

Reply:
Ahh also looks like azure-sru didn't even source this variable as it should. Fixing that now too